tests/installation/accept_license.pm

```diff
@@ -28,11 +28,11 @@ use warnings;
 use base 'y2_installbase';
 use y2_logs_helper qw(verify_license_has_to_be_accepted verify_license_translations accept_license);
 use testapi;
-use version_utils 'is_sle';
+use version_utils qw(is_sle is_microos);
 
 sub run {
     my ($self) = @_;
-    assert_screen('license-agreement', 120);
+    assert_screen('license-agreement', 120) unless is_microos;
```

Comment:
To get a better understanding how to handle needles O_o
wouldn't it be also possible to add a fitting needle named license-agreement, which then need to match here?

Reply:
AFAICT this just needs a new needle and the `unless is_microos` here is wrong.

Reply:
Ok. Just figured out that `welcome-no-product-list` needle also includes `license-agreement` tag, so I don't need to touch this line neither create a new needle. Will update PR.